dashboard/src/reducers/datasetListReducer.js

```diff
@@ -1,33 +1,64 @@
-import {
-  GET_PUBLIC_DATASETS,
-  FAVORITED_DATASETS,
-  UPDATE_PUBLIC_DATASETS,
-} from "../actions/types";
+import * as CONSTANTS from "assets/constants/browsingPageConstants";
+import * as TYPES from "actions/types";
 
 const initialState = {
   publicData: [],
   favoriteRepoNames: [],
   tableData: [],
+  offset: 0,
```

Comment:
Should this be `CONSTANTS.START_PAGE_NUMBER - 1`?

Reply:
I think you and I are in agreement on how the the offset gets calculated.  The question is, what should the contents of line 8 be?
- a literal zero?
- a symbolic constant whose value is zero? Or,
- an expression using an existing, related symbolic constant (which ends up being zero)?

It's not a huge deal.  It's just that, as a matter of general principal, we should avoid embedding "magic numbers" in the code, particularly if they have something to do with configuration or initial state which we could conceivably want to alter in the future.

Reply:
Updated the offset with constants